geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/CacheClientProxy.java

```diff
@@ -963,6 +968,10 @@ private void closeTransientFields() {
       return;
     }
 
+    closeOtherTransientFields();
```

Comment:
I wonder what block 965-967 would do. Can we always close all these transient fields no matter what?

Reply:
Yes, the transient fields should be closed (or cleaned up) when the CacheClientProxy closes.

They are mostly related to the socket. Transient fields are:

- _socket
- _commBuffer - ByteBuffer on the socket
- releaseResourcesForAddress - removes the asyncCloseExecutors for the proxy's _remoteHostAddress
- clearClientInterestList and closeNonDurableCqs - remove interests and CQs